### ISSUE TITLE
Add a definition/description field to Accessibility

### ISSUE THREAD

**Issue by Raphael-Gazzotti** [openMetadataInitiative/openMINDS_core#607]
The Accessibility library currently lacks clear guidance on the  usage of its different instances. Introducing this field would improve clarity and facilitate their correct use.

**Comment by Raphael-Gazzotti** [comment]
The use-case should be detailed in this property.
Other options: Purpose/Application/Utility/Intend

**Comment by Raphael-Gazzotti** [comment]
We will modify the name of this field depending on the text attached to the instances.
